.php_cs.dist

```diff
@@ -1,18 +1,29 @@
-<?php
+<?php declare(strict_types=1);
```

Comment:
could we please be consistent with the place where we add `declare(strict_types=1);`? Usually it's the third line from top.